docs/installation/Kepler-Telegraf-integration-steps.md

```diff
@@ -0,0 +1,467 @@
+
+Sample Prometheus configuration file is as follows:
+```
+# my global config
+global:
+scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+# scrape_timeout is set to the global default (10s).
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+# The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+	- job_name: 'kepler'
+	  static_configs:
+		- targets: ['xx.xx.xx:9102'] 
+	- job_name: 'telegraf'
+	  static_configs:
+		- targets: ['xx.xx.xx:9273']
+
+```
+Run the Prometheus container with the created Prometheus configuration file:
+```
+root@:~# docker run -d -p 9090:9090 -v $PWD/prometheus.yaml:/etc/prometheus/prometheus.yml prom/prometheus
+```
+On the Prometheus GUI at localhost:9090, it can be confirmed that Prometheus is scrapping from Kepler and Telegraf.
+
+![](../fig/Kepler-Telegraf-Prometheus.png)
+
+Download and start Grafana container
+==================================================
+
+Like, Prometheus, Grafana can be installed on the system in various
+ways. Here, we are installing Grafana's container image.
+```
+root@:~# docker run -d --network host --name grafana grafana/grafana
+```
+Once Grafana container is running access the Grafana GUI at localhost:3000. Login with default credentials. After login, The Prometheus database needs to be added as a data source into Grafana GUI. Click on “DATA SOURCES” -> “Add your first data source”
+And select Prometheus - > Click “Save and Test”
+
+Dashboard
+=======================
+
+Once Prometheus has been added as a data source, create a dashboard by
+exporting
+[Kepler-Exporter.json](https://github.com/sustainable-computing-io/kepler/blob/main/grafana-dashboards/Kepler-Exporter.json),
+a default Grafana dashboard for Kepler.
+
+This default Kepler dashboard can be edited in order to display Telegraf
+metric along with Kepler metrics.
+
+For example in below shown example, right hand shows Power related
+metrics collected by Telegraf whereas left hand shows Power related
+metrics by Kepler per namespace:
+
+![](../fig/Kepler-Telegraf-dashboard.png)
+
+**On Kepler side:**
+
+**PKG-\>** Represents kepler\_container\_package\_joules\_total metrics
+which measures the cumulative energy consumed by the CPU socket,
+including all cores and uncore components (e.g. last-level cache,
+integrated GPU and memory controller).
+
+**DRAM-\>** Represents kepler\_container\_dram\_joules\_total metric which
+describes the total energy spent in DRAM by a container.
+
+**Other-\>** Represents kepler\_container\_other\_joules\_total metric
+measures the cumulative energy consumption on other host components
+besides the CPU and DRAM. Generally, this metric is the host energy
+consumption (from acpi) less the RAPL Package and DRAM.
+
+**On Telegraf side:**
+
+
```

Comment:
Too many new lines

Reply:
Also maybe a bullet list for the items below would make it a little more readable.